sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionOwnership.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Messaging.EventHubs.Processor
+{
+    /// <summary>
+    ///   Contains all the information needed to describe the status of the owner of a partition.  It's used by
+    ///   <see cref="IPartitionManager" /> to claim ownership of a partition and to list existing ownership.
+    /// </summary>
+    ///
+    public class PartitionOwnership
+    {
+        /// <summary>
+        ///   The path of the specific Event Hub this partition ownership is associated with, relative
+        ///   to the Event Hubs namespace that contains it.
+        /// </summary>
+        ///
+        public string EventHubName { get; }
+
+        /// <summary>
+        ///   The name of the consumer group this partition ownership is associated with.
+        /// </summary>
+        ///
+        public string ConsumerGroup { get; }
+
+        /// <summary>
+        ///   The identifier of the associated <see cref="EventProcessor" /> instance.
+        /// </summary>
+        ///
+        public string InstanceId { get; }
```

Comment:
I'd recommend `Identifier` here for the reasons discussed in the `CheckpointManager` class.